fluent-bit/docker-compose.test.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+services:
+  fluent-bit:
+    build:
+      context: .
+      args:
+        FLUENTBIT_SUFFIX: -debug
+    image: chatwork/fluent-bit
+  sut:
+    image: kiwicom/dgoss
+    environment:
+      GOSS_FILES_PATH: /goss
+      GOSS_FILES_STRATEGY: cp
+    command: /usr/local/bin/dgoss run chatwork/fluent-bit /fluent-bit/bin/fluent-bit -c /fluent-bit/etc/fluent-bit.conf -f /dev/null
+    container_name: fluent-bit
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    depends_on:
+      - fluent-bit
```

Comment:
Since the fluent-bit image does not contain any sh, the debug image is used for testing.